aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -60,49 +59,48 @@ There are size-based, time-based, and group-based aggregators.
 </aggregators:size-based-aggregator>
 ----
 
-This example aggregates elements until a pre-defined size is reached, executing the routes and listeners.
+If the elements reach the *Max size* specified, two things occur:
+
+* The elements in the storage are removed and the next element will belong to the new aggregation.
+* The *Aggregation complete* route executes with the aggregated elements.
+
+Additionally, if an *Aggregator listener* is registered to the aggregator scope, the listener's callback executes with the same set of elements. +
+If the *Incremental aggregation* route is not null, and the *Max size* is not reached, then the route component chain executes with all the aggregated elements, including the last element aggregated.
+
+You can also define a *Timeout* for the aggregator scope. In that case, a scheduled task with that timeout as a delay is registered for execution. The time is computed from the time at which the first element arrives and no extra task is scheduled if there is another task waiting to be executed. In the case of a timeout, the referenced listener executes only if it supports being called by timeout.
 
 ==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Name | String | The name of the aggregator. It can later be referenced by an aggregator listener. || x
+| Name | String | The name of the aggregator. An aggregator listener can later reference the scope by name. || x
 | Content | Expression | The expression that defines what to aggregate. The result of the evaluation is the value stored in the aggregation. | #[payload] |
-| Max Size | Number | The total number of elements to be aggregated before considering the aggregation complete. | | x
-| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to max size, the aggregation is not considered complete. A value of `0` is not supported to avoid timing out the group constantly. | -1(UNLIMITED) |
+| Max Size | Number | The total number of elements to aggregate before considering the aggregation complete. | | x
+| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to Max Size, the aggregation is not considered complete. A value of `0` is not supported to avoid timing out the group constantly. | -1(UNLIMITED) |
```

Comment:
```suggestion
| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to Max Size, the aggregation is not considered complete. A value of `0` is not supported, to avoid constant group timeouts. | -1(UNLIMITED) |
```